src/components/layout/card/card.css

```diff
@@ -0,0 +1,13 @@
+.form__container {
+  background-color: var(--neutral-very-light);
+  margin: auto;
+  margin: 24px auto;
+  box-shadow: 0px 2px 5px 1px var(--neutral-dark);
+}
+
+.small {
+  width: 420px;
+}
```

Comment:
class `.small` is strictly related with `. form__container` and should be used as such. Your definition of small variant should look like this
```
.form__container.-small {
  width: 420px;
}
```

this prevents the styles overwriting when somewhere else you have defined class `small`

Reply:
CAUTION: there is no space between `.form__container` and `.-small` - this means "when one tag has those 2 classes"